src/main/resources/com/google/api/codegen/csharp/wrapper.snip

```diff
@@ -471,8 +462,22 @@
             @if param.isRepeated
                 {@param.propertyName} = { {@param.name} },
             @else
-                {@param.propertyName} = {@param.name},
+                {@param.propertyName} = {@param.name}{@param.propertyTransform},
             @end
         @end
     }
 @end
+
+@private pageStreamPartials(pages)
+    // Partial classes to enable page-streaming
+
+    @join page : pages
+        public partial class {@page.requestTypeName} : IPageRequest { }
```

Comment:
At some point we may need to do some work to make sure we handle requests/responses in a different namespace to the service, but we can punt for now.